boardgame-ui/src/locales/translations.csv

```diff
@@ -147,3 +147,22 @@ uservalidation.table.id,ID,
 uservalidation.table.username,Username,
 uservalidation.table.email,E-mail,
 uservalidation.table.validated,Validated ?,
+button.add-game,Add game,
+add-game.title,Add game,
+add-game.board-game.label,Board game,
+add-game.ranking-method.label,Ranking method,
+add-game.game-duration.label,Game duration,
+add-game.players.title,Players,
+add-game.board-game.no-result,No board game matching search,
+global.ranking-method.WIN_LOSE,Win/Lose,
+global.ranking-method.POINTS_LOWER_BETTER,Reverse ranking,
+global.ranking-method.POINTS_HIGHER_BETTER,Ranking,
+add-game.players.user,User,
+add-game.players.score,Score,
+add-game.players.winner,Winner,
+button.add-player,Add player,
+add-game.validation-error.board-game,This field should be filled with a valid board game,
+global.invalid-form,Form is not valid! Please check the fields.,
+add-game.save-error,Failed to save the game,
+add-game.save-success,The game was successfully added,
```

Comment:
May I suggest we say "successfully saved" as it is used for both Add and Update? :)